services/search/pkg/search/service_test.go

```diff
@@ -391,3 +391,51 @@ var _ = Describe("Searchprovider", func() {
 		})
 	})
 })
+
+var _ = DescribeTable("Parse Scope",
```

Comment:
A table test! \o/ ;)

Reply:
It would be awesome if we could add tests that prove that scopes are reflected in the `Ref.Path` field of the `SearchIndexRequests`.

Reply:
done